submarine-workbench/workbench-server/src/test/java/org/apache/submarine/database/service/TeamServiceTest.java

```diff
@@ -93,6 +93,7 @@ public void updateByPrimaryKeySelective() throws Exception {
     assertTrue(ret);
 
     team.setTeamName("submarine_update");
+    team.setUpdateBy("submarine_user_update");
     TeamMember teamMember_update = new TeamMember();
```

Comment:
Change `teamMember_update`  to `teamMemberUpdate`

Reply:
Done.